src/commands/init.js

```diff
@@ -146,6 +147,66 @@ async function init() {
 const sleep = (time: number = 1000) =>
   new Promise(resolve => setTimeout(resolve, time));
 
+const getRunScript = (scriptName: string) => {
+  const runCommand = scriptName === 'start' ? 'yarn' : 'yarn run';
+  return `${runCommand} ${scriptName}`;
+};
+
+/**
+ * Adds Haul to package.json scripts
+ */
+const addToPackageScripts = async (cwd: string) => {
+  const pjson = JSON.parse(
+    fs.readFileSync(path.join(cwd, 'package.json')).toString(),
+  );
+
+  const scripts = pjson.scripts || {};
+
+  const haulScript = Object.keys(scripts).find(name =>
+    scripts[name].includes('haul'));
+
+  if (haulScript) {
+    ora().info(
+      `Haul already exists in your package.json. Start Haul by running ${getRunScript(haulScript)}'`,
+    );
+    return;
+  }
+
+  let scriptName = 'start';
+
+  if (
+    scripts.start &&
+    scripts.start !== 'node ./node_modules/react-native/local-cli/cli.js start'
+  ) {
+    const result = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'scriptName',
+        message: 'Enter the name of the script to add to package.json',
+        default: scriptName,
```

Comment:
We should default to something else than `start`, I think `haul` is better, I think most of the time I'll go with this anyways